evap/evaluation/models.py

```diff
@@ -1185,6 +1188,9 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     # users to which all emails should be sent in cc without giving them delegate rights
     cc_users = models.ManyToManyField("UserProfile", verbose_name=_("CC Users"), related_name="ccing_users", blank=True)
 
+    # flag for proxy users like FSR or student office which are not actual users themselves, but represent a group of users
```

Comment:
"FSR" and "student office" are rather HPI specific. My last information is that in general, we want to use more wide range wording. @janno42?